API.md

```diff
@@ -63,19 +65,35 @@ Read the **[GETTING STARTED](https://github.com/orbitdb/orbit-db/blob/master/GUI
 
 <!-- tocstop -->
 
-## Constructor
+## Creating an OrbitDB instance
 
-### constructor(ipfs, [directory], [options])
+### createInstance(ipfs, [options])
 ```javascript
-const orbitdb = new OrbitDB(ipfs)
+const orbitdb = await OrbitDB.createInstance(ipfs)
 ```
-Creates and returns an instance of OrbitDB. Use the optional `directory` argument to specify a path to be used for the database files (Default: `'./orbitdb'`). In addition, you can use the optional `options` argument for further configuration. It is an object with any of these properties:
+Creates and returns an instance of OrbitDB. Use the optional `options` argument for further configuration. It is an object with any of these properties:
+
+- `directory` (string): path to be used for the database files. By default it uses `'./orbitdb'`.
 
 - `peerId` (string): By default it uses the base58 string of the ipfs peer id.
 
 - `keystore` (Keystore Instance) : By default creates an instance of [Keystore](https://github.com/orbitdb/orbit-db-keystore). A custom keystore instance can be used, see [this](https://github.com/orbitdb/orbit-db/blob/master/test/utils/custom-test-keystore.js) for an example.
 
-After creating an `OrbitDB` instance , you can access the different data stores. Creating a database instance, eg. with `orbitdb.keyvalue(...)`, returns a *Promise* that resolves to a [database instance](#store-api). See the [Store](#store-api) section for details of common methods and properties.
+- `identity` (Identity Instance): By default it creates an instance of [Identity](https://github.com/orbitdb/orbit-db-identity-provider/src/identity.js)
+
+### constructor(ipfs, identity, [options])
```

Comment:
I was thinking if we should leave out documenting the constructor. I believe what we want the users to do is to use `createInstance` instead of the constructor. If we document both, the user might think either one can be used and would prolly default to using the familiar one, the constructor.

Thoughts?

Reply:
I was thinking the same but thought perhaps we wanted them to use the constructor instead....this would however force users to create an identity first instead of having a default one created for them.

Reply:
Actually maybe it is better to have a default created since that should be what is used out of the box. Advanced users can create their own identities and pass it as an option. Agreed its more confusing to have another way to create an instance.